src/FabActUtil/Tool.cs

```diff
@@ -55,8 +55,20 @@ private static void GenerateOutput(ToolContext context)
             if (context.Arguments.Target == OutputTarget.Manifest)
             {
                 GenerateManifest(context);
-                AddParametersToLocalFiveNodeAppParamFile(context);
-                AddParametersToLocalOneNodeAppParamFile(context);
+
+                // If StartupServiceFilePath is provided, update parameters into service ParamFiles otherwise application ParamFiles
+                if (!string.IsNullOrEmpty(context.Arguments.StartupServicesFilePath))
+                {
+                    Console.WriteLine("Adding service parameters to service param files - {0} and {1}", context.Arguments.Local5NodeServiceParamFile, context.Arguments.Local1NodeServiceParamFile);

```

Comment:
I assume Local5NodeServiceParamFile contains filename not the file content. If not, I am not sure if we should be tracing all the file content.

Reply:
Got clarified by the below method. Its the filename.